smithy-jsonschema/src/main/java/software/amazon/smithy/jsonschema/JsonSchemaShapeVisitor.java

```diff
@@ -332,6 +332,14 @@ private Schema.Builder updateBuilder(Shape shape, Schema.Builder builder) {
         return builder;
     }
 
+    private Optional<String> descriptionMessage(Shape shape) {
+        String deprecatedMessageSuffix = shape.getTrait(DeprecatedTrait.class)
+                .map(deprecatedTrait -> deprecatedTrait.getDeprecatedDescription(shape.getType()))
+                .map(message -> "\n" + message).orElse("");
+        return shape.getTrait(DocumentationTrait.class).map(DocumentationTrait::getValue)
+                .map(documentation -> documentation + deprecatedMessageSuffix);
```

Comment:
```suggestion
        StringBuilder builder = new StringBuilder();
        shape.getTrait(DocumentationTrait.class).ifPresent(trait -> builder.append(trait.getValue());
        shape.getTrait(DeprecatedTrait.class).ifPresent(trait -> builder.append("\n").append(trait.getValue());
        return builder.toString().trim();
```

Also simpler with a `StringBuilder`.